src/v2/client/configIdxJsClient.js

```diff
@@ -21,5 +22,6 @@ export function configIdxJsClient(appState) {
     if (fingerprint) {
       requestConfig.headers['X-Device-Fingerprint'] = fingerprint;
     }
+    requestConfig.headers['X-Okta-User-Agent-Extended'] = `okta-signin-widget-${config.version}`;
```

Comment:
always set this ?

Reply:
Based on the description in the [Jira ticket](https://oktainc.atlassian.net/browse/OKTA-386994)

Reply:
Consider prepending to this header, if it already has a value mapped to it. This will allow us to pass through additional SDKs for better tracking: See #543

Reply:
looks like it's also missing from this wiki: https://oktawiki.atlassian.net/wiki/spaces/eng/pages/471335145/Technical+Design+for+User+Agents

For this PR, let's focus on make the okta-user-agent be consistent with V1, and take the rests as follow up.  
Actually, it's not even consistent for just V1, here is a tracking jira for that issue (https://oktainc.atlassian.net/browse/OKTA-401524).